ggml/experimental.py

```diff
@@ -971,6 +971,12 @@ def conv_1d_s1_ph(
         op = ggml.ggml_conv_1d_s1_ph(ctx.context, a.tensor, b.tensor)
         return Tensor(tensor=op, ctx=ctx)
 
+    @staticmethod
+    def conv_2d_sk_p0(a: "Tensor", b: "Tensor", ctx: Optional[Context] = None):
```

Comment:
nit: this can be changed to `def conv_2d_sk_p0(a: Tensor, b: Tensor, ctx: Optional[Context] = None):`